conflux-sponsor-faucet-sdk/faucet-frontend/faucet.js

```diff
@@ -54,15 +54,38 @@ class Faucet {
      * @param dapp The address of dapp
      */
     async apply(dapp) {
-        return await this.estimateForContract(this.faucet.applyBoth, [dapp]);
+        return await this.estimateForContract(this.faucet.applyGasAndCollateral, [dapp]);
     }
 
     /**
      * @dev check if a dapp can be sponsored
      * @param dapp The address of dapp 
      */
     async isAppliable(dapp) {
-        return await this.estimateForContract(this.faucet.isAppliable, [dapp]);
+        let r; 
+        try {
+            r = await this._isAppliableCall(dapp);
+            return r;    
+        } catch(e) {
```

Comment:
why catch exception here? _isAppliableCall will return the exception once failed.

Reply:
To get error message.